academic-requirements/src/components/FourYearPlanPage.tsx

```diff
@@ -16,6 +18,11 @@ const FourYearPlanPage = (props: {
     semesters: string;
     subject: string;
   }[];
+  TakenCourses: {
+    Course: string[];
+  }[];
+  MajorData: string;
+  ConcentrationData: string;
```

Comment:
This is just the name of the major and concentration, right? Could we possibly rename them to name instead of data just so we don't run into issues down the road of people thinking it's something else?

Reply:
Yeah that makes sense, I will get that changed.

Reply:
This is super picky, but while changing these, could make the variables camel case, just for consistency.

Reply:
Sure, I can do that as well.